newIDE/app/src/GameDashboard/GameCard.js

```diff
@@ -1,92 +1,263 @@
 // @flow
-import { Trans } from '@lingui/macro';
+import { t, Trans } from '@lingui/macro';
 import { I18n } from '@lingui/react';
-import { Card, CardActions, CardHeader, Chip } from '@material-ui/core';
+import { Card, Chip } from '@material-ui/core';
 import * as React from 'react';
 import { Column, Line, Spacer } from '../UI/Grid';
 import RaisedButton from '../UI/RaisedButton';
-import { getGameUrl, type Game } from '../Utils/GDevelopServices/Game';
+import {
+  getGameUrl,
+  updateGame,
+  type Game,
+} from '../Utils/GDevelopServices/Game';
 import { ResponsiveLineStackLayout } from '../UI/Layout';
 import Window from '../Utils/Window';
 import { GameThumbnail } from './GameThumbnail';
+import FlatButton from '../UI/FlatButton';
+import ShareIcon from '@material-ui/icons/Share';
+import IconButton from '../UI/IconButton';
+import Text from '../UI/Text';
+import Toggle from '../UI/Toggle';
+import MoreVert from '@material-ui/icons/MoreVert';
+import CardContent from '../UI/Card/CardContent';
+import AuthenticatedUserContext from '../Profile/AuthenticatedUserContext';
+import CardHeader from '../UI/Card/CardHeader';
+import ElementWithMenu from '../UI/Menu/ElementWithMenu';
+import { type I18n as I18nType } from '@lingui/core';
+import { type GamesDetailsTab } from './GameDetailsDialog';
+import { useDebounce } from '../Utils/UseDebounce';
+import ShareDialog from './ShareDialog';
```

Comment:
As a nitpicking:
I like to have imports organized so that they look like:
```
Usual imports (react, i18n)
Material icons
UI components
Other
```
Separated by new lines. It can help sometimes understand the context of the component (Is is really dependant on Material ? Is it self-sufficient or does it use a lot of sub components? etc.

But this is not mandatory

Reply:
I like this, it looks more organised and clean! :D